src/docs/reference/support-policy.md

```diff
@@ -60,6 +60,37 @@ In the example above, when Version D is released, Version C enters maintenance m
 After July 1st 2022, Version C will be in extended support until Jun 1st, 2023, twelve months after the release of
 Version D.
 
+## TypeScript Support
+
+Stencil acts as a compiler for a project's web components, and works closely with the TypeScript compiler to transform
+TSX to vanilla JavaScript. To ensure compatibility between the two, Stencil takes an opinionated stance on which version
+of the TypeScript compiler that must be used.
+
+Stencil includes a recent copy of the TypeScript compiler in its distributable* to guarantee this compatibility. 
+The Stencil team is committed to keeping its version of TypeScript up to date, and as of Stencil v2.10.0 attempts to be
```

Comment:
Not a grammar expert, but I think this may be correct

```suggestion
The Stencil team is committed to keeping its version of TypeScript up to date and, as of Stencil v2.10.0, attempts to be
```